GetJobber/atlantis
# feat(components-native): Migrate format file

## Motivations



## Changes



### Added

- components-native: Added `FormatFile`

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- In Jobber Mobile checkout `test-format-file`
- Run a local build
- Verify FormatFile works on Android and iOS (All of them have been replaced with this new implementation)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
